templates/deployments.yaml

```diff
@@ -973,9 +976,12 @@ spec:
         chart: {{ .Chart.Name }}-{{ .Chart.Version }}
         release: {{ .Release.Name }}
         heritage: {{ .Release.Service }}
+      {{- with .Values.st2actionrunner.annotations }}
```

Comment:
By using `with` here the bracket will hide the required `checksum/config` and `checksum/ssh` annotations if st2actionrunner annotations is empty.

Reply:
Solved